docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md

```diff
@@ -84,11 +82,11 @@ Now you have retrieved the relevant documentation, you can package content with
 
 The values from the CPIX documents will need to be copied and pasted to the corresponding section in the examples below.
 
-For more information, please refer to the Shaka Packager Documentation which can be found at the following link - [Shaka Packager Documentation](https://google.github.io/shaka-packager/html/)
+For more information, please refer to the documentation from [Shaka Packager](https://google.github.io/shaka-packager/html/).
 
-### Packaging Content with Widevine and Playready DRM
+### Packaging Content with Widevine and PlayReady DRM
 
-Below is an example of how we can send a request to Shaka Packager to encrypt our content with Widevine and Playready DRM and package it.
+Below is an example of how we can send a request to Shaka Packager to encrypt our content with Widevine and PlayReady DRM and package it.
```

Comment:
```suggestion
Below is an example of how we can send a request to Shaka Packager to encrypt and package our content with Widevine and PlayReady DRM.
```
Slight rewording, as the sentence felt like a bit of a mouth full.

Reply:
Actioned